api/v1beta1/keystoneendpoint.go

```diff
@@ -190,3 +190,52 @@ func DeleteKeystoneEndpointWithName(
 
 	return nil
 }
+
+// GetKeystoneEndpointList func
+func GetKeystoneEndpointList(
+	ctx context.Context,
+	h *helper.Helper,
+	namespace string,
+) (*KeystoneEndpointList, error) {
+
+	ke := &KeystoneEndpointList{}
+	err := h.GetClient().List(ctx, ke)
```

Comment:
its not filtered for the namespace, or ? should probably use some list option:

```
        listOpts := []client.ListOption{
                client.InNamespace(namespace),
        }

        if err := h.GetClient().List(ctx, ke, listOpts...); err != nil {
                err = fmt.Errorf("Error listing endpoints for namespace %s: %w", namespace, err)
		return nil, err
        }
```

Reply:
Absolutely right, sir.